LocalWiki.php

```diff
@@ -182,6 +182,14 @@ function wfDonateLink( $skin, &$bar ) {
 	unset( $wgGroupPermissions['checkuser'] );
 }
 
+if ( $wgDBname === 'snapwikiwiki' ) {
```

Comment:
I'm pretty sure that these can rather be added to LocalSettings.php rather then here. Thanks!

Reply:
You will have to wait for a few hours then.